### DISCUSSION TITLE
Custom Features for FeatureExtractor

### DISCUSSION THREAD

**Discussion by koaning** [explosion/spaCy#7279] [Help: Other Questions] [answered]
When I look at the settings for the [FeatureExtractor](https://spacy.io/api/architectures#FeatureExtractor) I see an example config listed. 

```yml
[model]
@architectures = "spacy.FeatureExtractor.v1"
columns = ["NORM", "PREFIX", "SUFFIX", "SHAPE", "ORTH"]
```

It seems to encode features that are attached to `Token` objects. What if I'd like to add my own features to the mix here? It's recommended to add features using an extension. Is it also possible to refer to features here that were added from an extension?

**Comment by adrianeboyd** [comment, marked as answer]
Sure, with a custom feature extractor. Here's a user's example: https://github.com/explosion/spaCy/discussions/6527#discussioncomment-186516
